PurchasesTests/Resources/receipts/verifyReceiptSample1.txt

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
this isn't used, but it's a useful reference when writing tests for the other sample, since it has the `/verifyReceipt` version of the same file